### DISCUSSION TITLE
[BETA] Userscripts v4.2.4 (macOS), v1.2.4 (iOS)

### DISCUSSION THREAD

**Discussion by quoid** [quoid/userscripts#311] [Announcements]
## Build 60/33

- [macOS/iOS] - fixed bug that prevented US.closeTab() API method from working - #310

**Comment by DBa2016** [comment]
@quoid Is there any way to get on the Beta list? I filled out the form but never got a response...

**Reply by quoid** [reply]
Shoot me an email (it's on my profile page) - I will make sure you are/get added

**Reply by DBa2016** [reply]
Done :)

**Comment by quoid** [comment]
rolled into: https://github.com/quoid/userscripts/discussions/346
